source/configuration/environments/generic/networking.rst

```diff
@@ -111,3 +111,36 @@ are available, we recommend using the DNS servers from `Quad9 <https://www.quad9
      - 9.9.9.9
      - 149.112.112.112
    resolvconf_search: osism.io
+
+
+Network Interface Card Offloading
+=================================
+
+If vxlan or geneve tunnel's for overlay networks enabled. For performance issues
```

Comment:
```
This section only applies if vxlan or geneve tunnels are enabled for overlay networks. To avoid performance issues, you can enable hardware offloading.
```

Maybe this sounds a little bit smoother.